core/api/src/graphql/public/root/mutation/support-chat-message-add.ts

```diff
@@ -0,0 +1,58 @@
+import { SupportChat } from "@/app"
+
+import { mapAndParseErrorForGqlResponse } from "@/graphql/error-map"
+import { GT } from "@/graphql/index"
+import IError from "@/graphql/shared/types/abstract/error"
+import SupportMessage from "@/graphql/public/types/object/support-message"
+
+const SupportChatMessageAddInput = GT.Input({
+  name: "SupportChatMessageAddInput",
+  fields: () => ({
+    message: { type: GT.NonNull(GT.String) },
+  }),
+})
+
+const SupportChatMessageAddPayload = GT.Object({
+  name: "SupportChatMessageAddPayload",
+  fields: () => ({
+    errors: {
+      type: GT.NonNullList(IError),
+    },
+    supportMessage: {
+      type: GT.List(SupportMessage),
+    },
+  }),
+})
```

Comment:
uhmm should we move this to payloads folder?

Reply:
I've been thinking about it, I actually think we should get rid of the payload folder, because there is a one to one relation between mutation and payload, and it doesn't make a lot of sense that we have to add/delete/open 2 files when working on a mutation

I'll do this general change in another PR if that's fine with you

Reply:
Strongly agree - locality of information is key! Group by concept (SupportChartMessage) not by role (XxxPayload).